### ISSUE TITLE
Insufficient color contrast for standard text

### ISSUE THREAD

**Issue by jrbenn** [strategicdata/pmhc-mds-user-documentation#17]
Issues # 18 in https://docs.google.com/spreadsheets/d/13iTb4aW7KulDcbj5Q2xD78H4KOrMmhvfIDW1JMffY7o/edit?gid=155984173#gid=155984173
